shuup/front/forms/product_list_modifiers.py

```diff
@@ -371,6 +371,12 @@ def get_filters(self, request, data):
             categories = list(categories)
 
         categories = [cat.strip() for cat in categories if cat]
+        category_objects = Category.objects.filter(pk__in=categories)
+
+        descendants = [list(dec.get_descendants().values_list("pk", flat=True)) for dec in category_objects]
```

Comment:
I would double think the performance here.

Reply:
Agreed, I think there is a way of doing this in a single queryset. You need to understand the MPTT model so you can filter properly. Please check this one: https://github.com/django-mptt/django-mptt/blob/master/mptt/managers.py#L191 
Test that you can use this way:
```py
categories = Category.objects. get_queryset_descendants(Category.objects.filter(pk__in=categories), include_self=True)
```